hooks/post/30-upgrade.rb

```diff
@@ -38,6 +38,14 @@ def import_subscriptions
   Kafo::Helpers.execute('foreman-rake katello:upgrades:2.4:import_subscriptions')
 end
 
+def remove_elasticsearch
+  gems = ['ruby193-rubygem-tire', 'tfm-rubygem-tire', 'elasticsearch', 'sigar', 'lucene3']
+  gems.each do |gem|
+    Kafo::Helpers.execute('rpm -e #{gem}')
+  end
+  echo "Remove elastic search files with #rm -rf /var/lib/elasticsearch"
```

Comment:
I think this is a good spot for it, however, I'd give them more information here. Something explaining that Elasticsearch has been removed as a dependency but the user needs to manually remove the database files and that they can do so via your command listed.